packages/fluent/scss/slider/index.scss

```diff
@@ -0,0 +1,26 @@
+// Module meta
+$_kendo-module-meta: (
+    name: "slider",
+    dependencies: (
+        "icon",
+        "button"
+    )
+);
+
+
+// Component
+@forward "_variables.scss";
+@use "_layout.scss" as *;
+@use "_theme.scss" as *;
+
+// Register
+@use "../core/module-system" as module;
```

Comment:
shall we add / at the end of module-system here (as per this PR: https://github.com/telerik/kendo-themes/pull/3795/files )